src/components/ec-table-head/ec-table-head.vue

```diff
@@ -15,8 +15,14 @@
         :colspan="column.span"
         scope="col"
       >
-        <span class="ec-table-head__cell-wrapper">
-          <span>
+        <span
+          class="ec-table-head__cell-wrapper"
+          :class="{
+            'ec-table-head__cell-wrapper--text-center': column.type === 'icon',
```

Comment:
change it please to { [`ec-table-head__cell-wrapper--is-type-${column.type}`]: !!column.type }

Reply:
I love this way of defining the classes according to the type, I think it's much more smart and reusable, but don't you think it can give us problems when we eventually want to include the plugin to purge css with Tailwind?